srtcore/buffer.h

```diff
@@ -428,7 +440,7 @@ class CRcvBuffer
        return m_iStartPos == m_iLastAckPos;
    }
    bool full() const { return m_iStartPos == (m_iLastAckPos+1)%m_iSize; }
-   int capacity() const { return m_iSize; }
+   int capacity() const { return m_iSize; } // XXX WHY it was changed to m_iSize-1 ?
 
```

Comment:
```
// XXX WHY it was changed to m_iSize-1 ?
```
What does this comment refer to?

Reply:
Let it stay here. There's some confusion I found in the changes for the groups; I found no real reason to make any change after I refreshed that, but this is simply left a comment for further investigation.

Reply:
Almost missed this. The comment itself is not clear it you want it to stay.
" WHY it was changed to m_iSize-1 ?"
It is now just "m_iSize". Where was it changed?
Maybe you meant " WHY m_iSize-1 was changed to m_iSize?"

Reply:
Ok, I'll try to make it more meaningful.

Reply:
Ok, I think I solved this mystery. This function is used exclusively in logs and nowhere else. Probably then I made this return size-1 and removed the `-1` extra added to the result everywhere where this function is used. This can be then forgotten.